packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -524,12 +614,21 @@ function useMediaUploadQueue() {
           item.originalResourceId === resourceId
       );
 
+    /**
+     * Whether a given resource can be transcoded.
+     *
+     * Checks whether the resource is not external and
+     * not already uploading.
+     *
+     * @param {import('@web-stories-wp/media').Resource} resource Resource object.
+     * @return {boolean} Whether a given resource can be transcoded.
+     */
     const canTranscodeResource = (resource) => {
       const { isExternal, id, src } = resource || {};
+
       return (
         !isExternal &&
         src &&
-        !isBlobURL(src) &&
```

Comment:
Why are you removing this?

Reply:
Because it's redundant. 

If it's a blob URL then the resource **must** be `isCurrentResourceProcessing` or `isResourceProcessing` already.

There's no need to check for blob URLs. We only must check on what's in the queue.